docs/integrating-with-unity/defining-states.mdx

```diff
@@ -0,0 +1,334 @@
+---
+sidebar_label: 'Defining States'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Defining States
+
+:::info
+TODO: We need to write a better description about why it is important to limit the state exposed to your bot.
+If we can't justify this vs accessing game object state directly, we should question what value we add...
+:::
+
+The state of the game sent to Regression Games is limited to only GameObjects selected to be included as part of the state.
+
+**A GameObject can be included as part of the state using one or more of the following methods.**
+
+## 1 - `RGStatefulGameObject` MonoBehaviour
+By adding the `RGStatefulGameObject` behaviour to a GameObject, that GameObject will be included in the state.
+Unless one of the other methods is used, this will include only the 'core' state properties of the GameObject.
```

Comment:
Might be good to define what the 'core' is here.

Reply:
I see now that is below - maybe we bring that up here or at least make the word 'core' a link to the section below.

Reply:
Actually, maybe disregard... in Markdown it seems far away, but any reader on the site should see the next section pretty easily as they read this.

Reply:
all references to core are now links to the core state properties table